Swift-Base/Scripts/mogen.sh

```diff
@@ -13,7 +13,7 @@
         #"${SRCROOT}/ios-base/Scripts/mogen.sh"
 #  Set Output files = $(DERIVED_FILE_DIR)/${INPUT_FILE_BASE}.mom
 
-mogenerator --model "${INPUT_FILE_PATH}" --machine-dir "${PROJECT_DIR}/ios-base/Models/CoreData/Private/" --human-dir "${PROJECT_DIR}/ios-base/Models/CoreData/" --template-var arc=true
```

Comment:
I think the file we can delete. Because it doesn't use. We use only mogend.sh script https://github.com/fs/ios-base-swift/pull/16/files#diff-e2314b95558cd9d65f822b4538f917c4R38 and https://github.com/fs/ios-base-swift/pull/16/files#diff-e2314b95558cd9d65f822b4538f917c4R48.